jina/serve/networking.py

```diff
@@ -86,9 +86,16 @@ def add_connection(self, address: str):
             self._address_to_channel[address] = channel
             self._connections.append(stubs)
 
-    async def remove_connection(self, address: str) -> Union[grpc.aio.Channel, None]:
+    async def _remove_connection(self, address: str) -> Union[grpc.aio.Channel, None]:
```

Comment:
This is not private, since it is called from a different class, we cannot just decide to make it private because we want so